packages/make-styles/src/constants.ts

```diff
@@ -8,4 +8,4 @@ export const RTL_PREFIX = 'r';
 
 export const SEQUENCE_PREFIX = '__';
 
-export const DEFINITION_LOOKUP_TABLE: Record<string, MakeStylesMatchedDefinitions> = {};
+export const DEFINITION_LOOKUP_TABLE: Record<string, [MakeStylesMatchedDefinitions, boolean /* isRTL */]> = {};
```

Comment:
`DEFINITION_LOOKUP_TABLE` now knows about definitions' text direction, it works because each key in it is a hash and it's unique for LTR/RTL.